runtime/bifrost/src/lib.rs

```diff
@@ -1987,6 +1987,34 @@ impl_runtime_apis! {
 	}
 }
 
+pub struct CustomOnRuntimeUpgrade;
+impl OnRuntimeUpgrade for CustomOnRuntimeUpgrade {
+	#[cfg(feature = "try-runtime")]
+	fn pre_upgrade() -> Result<(), &'static str> {
+		#[allow(unused_imports)]
+		use frame_support::{migration, Identity};
+		log::info!("Bifrost `pre_upgrade`...");
+		Ok(())
+	}
+
+	fn on_runtime_upgrade() -> Weight {
+		log::info!("Bifrost `on_runtime_upgrade`...");
+		Salp::set_multisig_account(ConfirmMuitiSigAccount::get());
+		SalpLite::set_multisig_account(PolkaConfirmAsMultiSig::get());
+		log::info!("Bifrost `on_runtime_upgrade finished`");
+		RocksDbWeight::get().writes(1)
+	}
+
+	#[cfg(feature = "try-runtime")]
+	fn post_upgrade() -> Result<(), &'static str> {
+		let mut account = Salp::multisig_confirm_account();
+		log::info!("after salp migration multisig account as {:?}", account);
+		account = SalpLite::multisig_confirm_account();
+		log::info!("after salp-lite migration multisig account as {:?}", account);
+		Ok(())
+	}
+}
+
```

Comment:
`CustomOnRuntimeUpgrade` is not used, remove this section.

Reply:
not enable in [executor](https://github.com/bifrost-finance/bifrost/blob/755fe00a13ba00947e108095056fb54ab2f60871/runtime/bifrost/src/lib.rs?_pjax=%23js-repo-pjax-container%2C%20div%5Bitemtype%3D%22http%3A%2F%2Fschema.org%2FSoftwareSourceCode%22%5D%20main%2C%20%5Bdata-pjax-container%5D#L1689) and  would suggest we reserve the template codes